### DISCUSSION TITLE
Curtain motor TS0601 _TZE200_clm4gdw4

### DISCUSSION THREAD

**Discussion by Lechuuuuuu** [Koenkk/zigbee2mqtt#19635] [Devices]
I am running Zigbee2MQTT on Home Assistant ( latest version ). it is possible to add this device ? 
I suppose that is the same as other covers [ LINK ](https://www.zigbee2mqtt.io/devices/TS0601_cover_1.html) it may work. Anybody has tried this model before?

**Comment by ajgon** [comment]
For anyone stumbling into this, and ending here from google - this definition should do the trick:

```js
const {} = require('zigbee-herdsman-converters/lib/modernExtend');
const exposes = require('zigbee-herdsman-converters/lib/exposes');
const e = exposes.presets;
const ea = exposes.access;
const tuya = require('zigbee-herdsman-converters/lib/tuya');

const definition = {
    fingerprint: [
        {
            modelID: 'TS0601',
            manufacturerName: '_TZE200_clm4gdw4',
        },
    ],
    model: 'Z100',
    vendor: 'LV-Tron',
    description: 'Smart Curtain Motor',
    fromZigbee: [tuya.fz.datapoints],
    toZigbee: [tuya.tz.datapoints],
    onEvent: tuya.onEventSetTime,
    configure: tuya.configureMagicPacket,
    exposes: [
      e.cover_position().setAccess('position', ea.STATE_SET),
      e.enum('motor_direction', ea.STATE_SET, ['normal', 'reversed']).withDescription('Set the motor direction'),
    ],
    meta: {
        tuyaDatapoints: [
          [1, 'state', tuya.valueConverterBasic.lookup({'OPEN': tuya.enum(2), 'STOP': tuya.enum(1), 'CLOSE': tuya.enum(0)})],
          [2, 'position', tuya.valueConverter.coverPosition],
          [3, 'position', tuya.valueConverter.raw],
          [5, 'motor_direction', tuya.valueConverterBasic.lookup({'normal': false, 'reversed': true})],
        ],
    },
};

module.exports = definition;
```

**Reply by Koenkk** [reply]
Does it have a battery and if yes, does it work when adding `[101, 'battery', tuya.valueConverter.raw],` and `e.battery()` to exposes?

**Reply by ajgon** [reply]
Mine particular models are powered from wall socket. Not sure if there is a battery-powered version with exact same `manufacturerName`.

**Reply by Koenkk** [reply]
Added it, thanks!

Could you also submit a picture for the docs? This can be done by:
1. Create a fork by clicking [here](https://github.com/Koenkk/zigbee2mqtt.io/fork)
2. Go to the `public/images/devices` directory, *Add file* -> *Upload files*
3. Upload the files and press *Commit changes*
4. Press *Contribute* -> *Open pull request* -> update title/description -> *Create pull request*

Make sure that:
- The filename is `TS0601_cover_10.png`
- The size is 512x512
- The background is transparent (use e.g. [Adobe remove background](https://new.express.adobe.com/tools/remove-background))

**Comment by ziul03** [comment]
working for modelID: 'TS0601',
            manufacturerName: '_TZE200_txrzy8hs',
            thanks
